style.css

```diff
@@ -0,0 +1,93 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    color: #6c757d;
+    font-family: "Segoe UI";
+    background-color: #efefef;
+}
+
+.main {
+    margin: 0 auto;
+    text-align: center;
+    line-height: 20px;
+    padding: 30px;
+}
+
+.main h1 {
+    font-weight: 400;
+}
+
+.main strong {
+    color: #0e0e0e;
+    font-weight: 500;
+    margin-bottom: 10px;
+}
+
+.container {
+    margin: 0 100px;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 50px;
+}
+
+.container h3 {
+    color: #0e0e0e;
+}
+
+.box {
+    background-color: white;
+    border-radius: 5px;
+    padding: 20px;
+    box-shadow: 0 5px 10px 1px #728487ba;
```

Comment:
This is cool, I didn't know how to do this and it looks much better/closer to the screenshot we had to copy with the box shadow on. Thanks for teaching me something :)

Reply:
I'm glad it helped you.